### DISCUSSION TITLE
Getting Voxelab Aquila boards to work

### DISCUSSION THREAD

**Discussion by classicrocker883** [mriscoc/Ender3V2S1#479] [General]
Has anyone tried this firmware with either the G32 or N32 chips for the Voxelab Aquila?

Ive been able to get Manual Mesh to work with ProUIex enabled. However, Ive got UBL to work only with that disabled.

but, Ive been able to get Auto Bed Leveling Bilinear to work with ProUIex enabled, however the BL/CR touch wont deploy. it does pass the initial self-test when first starting, but will not deploy when selected in the menu.

I think there is a problem with BL touch and ProUIex why they aren't exactly compatible.

The most current source code wont allow a good flash to the board, but the previous version stm32f1-maple.ini file did allow it to work.

also, I get a white screen and some icon artifacts when first flashing the firmware, this goes away upon restarting.

**Comment by mriscoc** [comment]
Maybe some of the recents changes in Marlin trigger that bug, could be interesting revert to the previous version and compare changes. I don't have a Voxelab printer to do tests.
